locales/fr.json

```diff
@@ -1,23 +1,23 @@
 {
   "command.config_display_language": "Changer la langue affichée",
-  "command.config_locales": "Configurer manuellement le chemin de la locale",
-  "command.config_locales_auto": "Détecter automatiquement le chemin de la locale",
+  "command.config_locales": "Configurer manuellement le chemin de la traduction",
+  "command.config_locales_auto": "Détecter automatiquement le chemin de la traduction",
   "command.config_source_language": "Changer la langue source",
   "command.copy_key": "Copier une Clé i18n",
   "command.deepl_usage": "Utilisation de l'API DeepL ",
   "command.delete_key": "Supprimer une Clé",
   "command.duplicate_key": "Dupliquer une Clé",
   "command.edit_key": "Modifier la traduction",
-  "command.extract.disable-auto-detect": "Désactiver la détection automatique du texte écrit en dur",
-  "command.extract.enable-auto-detect": "Activer la détection automatique du texte écrit en dur",
+  "command.extract.disable-auto-detect": "Désativer l'auto-détection des chaines codées en dur.",
+  "command.extract.enable-auto-detect": "Valider l'autodétection des chaines codées en dur.",
   "command.fulfill_keys": "Remplir les champs vides de Clés",
-  "command.go_to_next_usage": "Aller à l'utilisation suivante",
-  "command.go_to_prev_usage": "Aller à l'utilisation précédente",
+  "command.go_to_next_usage": "Passer à l'utilisation suivante",
+  "command.go_to_prev_usage": "passer à l'utilisation précédente",
```

Comment:
Do we have "passer" in a lower-case on purpose?

Reply:
No it's a mistake. Passer is better.

Reply:
+1